habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml

```diff
@@ -141,7 +141,7 @@ habitat:
       agent_0:
         joint_start_noise: 0.0
   dataset:
-    data_path: data/datasets/manipulation/manipulation_eval.json.gz
```

Comment:
Have you updated the data locally? I don't have manipulation_eval_test.json.gz .

Reply:
I you do want to change the data-related config,  please upload your data to google drive for the reviewer to test.

Reply:
Solved. I will remember to upload data for test the next time. Thank you for your kind advice!

Reply:
Please also change the dataset explanation in the habitat-mas remade accordingly. 

https://github.com/SgtVincent/habitat-lab/blob/cae45df26fdc3ae25e4093f5762007dcb9c89d00/habitat-mas/README.md#L63-L92

Reply:
> Please also change the dataset explanation in the habitat-mas remade accordingly.
> 
> https://github.com/SgtVincent/habitat-lab/blob/cae45df26fdc3ae25e4093f5762007dcb9c89d00/habitat-mas/README.md#L63-L92

Done. The new datasets is placed in data/datasets/manipulation/manipulation_eval.json.gz, so the datasets explanation could remain the same. Nothing has to be changed.